site/src/pages/RoadmapPage/CourseHitItem.tsx

```diff
@@ -19,7 +20,7 @@ const CourseHitItem: FC<CourseHitItemProps> = (props: CourseHitItemProps) => {
     dispatch(setActiveCourse(props));
     dispatch(setShowAddCourse(true));
     // also hide the search bar to view the roadmap
-    dispatch(setShowSearch(false));
+    dispatch(setShowSearch({ show: false }));
```

Comment:
Should be removed so that the search doesn't close when you click on a course and you can go back if you cancel adding a course. I think it was like that earlier in the PR but reverted after merging from main.
```suggestion
```